pico/api/src/main/java/io/helidon/pico/api/ServiceInfoBasics.java

```diff
@@ -32,10 +32,11 @@
 public interface ServiceInfoBasics {
 
     /**
-     * Default weight for any weighted component (whether it implements this interface
-     * or uses {@link  io.helidon.common.Weight} annotation).
+     * Default weight for any <i>internal</i> Pico service component. It is defined to be
+     * {@link Weighted#DEFAULT_WEIGHT} {@code - 1} in order to allow any other service implementation to
+     * naturally have a higher weight (since it will use the {@code DEFAULT_WEIGHT} unless explicitly overridden.
      */
-    double DEFAULT_WEIGHT = Weighted.DEFAULT_WEIGHT;
+    double DEFAULT_PICO_WEIGHT = Weighted.DEFAULT_WEIGHT;
```

Comment:
I do not see the `-1` mentioned in the javadoc

Reply:
Great catch! lol

Reply:
pushed